doc/release-notes-1.1.0.rst

```diff
@@ -0,0 +1,621 @@
+See :ref:`Getting the nRF Connect SDK code <cloning_the_repositories_win>` for more information.
+
+
+Supported modem firmware
+************************
+
+* mfw_nrf9160_1.1.0
+
+Use the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See `Updating the nRF9160 DK cellular modem`_ for instructions.
+
+
+Supported boards
+****************
+
+* PCA10090 (nRF9160 DK)
+* PCA20035 (Thingy:91)
+* PCA10056 (nRF52840 Development Kit)
+* PCA10059 (nRF52840 Dongle)
+* PCA10040 (nRF52 Development Kit)
+* PCA10028 (nRF51 Development Kit)
+* PCA63519 (Smart Remote 3 DK add-on)
+
+
+Required tools
+**************
+
+In addition to the tools mentioned in :ref:`gs_installing`, the following tool versions are required to work with the |NCS|:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Tool
+     - Version
+     - Download link
+   * - SEGGER J-Link
+     - V6.41
+     - `J-Link Software and Documentation Pack`_
+   * - nRF Command Line Tools
+     - v9.8.1
+     - `nRF Command Line Tools`_
+   * - nRF Connect for Desktop
+     - v3.0.0 or later
+     - `nRF Connect for Desktop`_
+   * - dtc (Linux only)
+     - v1.4.6 or later
+     - :ref:`gs_installing_tools_linux`
+   * - GCC
+     - See :ref:`Installing the toolchain<gs_installing_toolchain_win>`
+     - `GNU Arm Embedded Toolchain`_
+
+
+As IDE, we recommend to use |SES| (Nordic Edition), version 4.20a or later.
+It is available from the following links:
+
+* `SEGGER Embedded Studio (Nordic Edition) - Windows x86`_
+* `SEGGER Embedded Studio (Nordic Edition) - Windows x64`_
+* `SEGGER Embedded Studio (Nordic Edition) - Mac OS x64`_
+* `SEGGER Embedded Studio (Nordic Edition) - Linux x86`_
+* `SEGGER Embedded Studio (Nordic Edition) - Linux x64`_
+
+
+Change log
+**********
+
+The following sections provide detailed lists of changes by component.
+
+
+???
+===
+
+(? where should these go? where are they defines/documented? what do they do?)
+
+(new) PAW3212 driver
+(new) nrf52810_pca20045 board
+(new) nrf52_pca20044 board
+
+adp536x driver:
+
+    Added buck discharge resistor configuration.
+
+
+
+nRF9160
+=======
+
+* Added the following samples:
+
+  * :ref:`lwm2m_carrier` - demonstrates how to use the :ref:`liblwm2m_carrier_readme` library to connect to the operator LwM2M network.
+  * :ref:`lwm2m_client` - demonstrates how to use LwM2M to connect an nRF9160 DK to an LwM2M server such as Leshan Demo Server via LTE.
+  * :ref:`usb_uart_bridge_sample` - acts as a serial adapter for Thingy:91, exposing 2 UART pairs to a USB host as 2 CDC ACM devices.
+
+
+* Added the following libraries:
+
+  * Cloud API (``include/net/cloud.h``) - provides a generic cloud API with an implementation for the nRF Cloud.
+  * :ref:`liblwm2m_carrier_readme` (version 0.8.0) - provides support for the Verizon Wireless network support.
+
+* Added the following drivers:
+
+  * nRF9160 GPS - (? where is that? what does it do?)
```

Comment:
?

Reply:
It's [this driver](https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/master/drivers/nrf9160_gps/nrf9160_gps.c) introduced by https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1029.

It configures GPS in the modem and controls GPS data readout from the modem.

Reply:
@jtguggedal Maybe you should have a look at this.

Reply:
It's  as @rlubos says. The driver configures the modem for GPS operation, and controls the GPS data coming from the modem. Applications must interact with the GPS using the GPS API and not the driver directly, as it has no public header. The driver is used by the asset tracker. Note that the driver is marked experimental here:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/81646300ceb6aa36faabbb5518075d6cec35ce68/drivers/nrf9160_gps/Kconfig#L9

Reply:
Sorry about the late response, @ru-fu , I overlooked this one.

Reply:
Updated to:
  * nRF9160 GPS (``drivers/nrf9160_gps/``) - configures the modem for GPS operation and controls the GPS data coming from the modem.
    Applications must interact with the GPS using the GPS API (``includ/gps.h``) and not use the driver directly.

@bjorn-tore-taraldsen @shantha-14 Should we mark it as experimental?

Reply:
No I don't think so. (We do not that for other modules)

Reply:
ok